benchmark/utils/DriverUtils.swift

```diff
@@ -268,7 +298,11 @@ func runBench(_ name: String, _ fn: (Int) -> Void, _ c: TestConfig) -> BenchResu
     var elapsed_time : UInt64 = 0
     if c.fixedNumIters == 0 {
       elapsed_time = sampler.run(name, fn: fn, num_iters: 1)
-      scale = UInt(time_per_sample / elapsed_time)
+      if elapsed_time > 0 {
```

Comment:
I think swift would have caused this otherwise to crash. I am not sure what behavior is the *correct behavior* though.

I need to think about it more, but I imagine in the case where elapsed_time is 0 says that something wrong has occurred and we probably want to notify the user in some way. Not with a crash ; ). But maybe with a warning? Not sure. What do you think @mtake?